api/DHSC.FingertipsNext.Modules.Core.Repository/HealthMeasure.cs

```diff
@@ -0,0 +1,23 @@
+﻿using DHSC.FingertipsNext.Modules.Repository.Dimensions.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DHSC.FingertipsNext.Modules.Repository
+{
+    [Serializable]
+    public sealed class HealthMeasure
+    {
+        public required AreaDimension AreaDimension { get; set; }
+        public required IndicatorDimension IndicatorDimension { get; set; }
+        public required SexDimension SexDimension { get; set; }
```

Comment:
The different dimensions may or may not exist or apply to particular health measures.  Are you sure all the properties should be 'required'.

Reply:
The underlying schema has everything as `NOT NULL` but I'll check if that's necessary.